Vungle/VungleRouter.m

```diff
@@ -14,6 +14,7 @@
 #endif
 #import "VungleInstanceMediationSettings.h"
 #import "VungleAdapterConfiguration.h"
+#import <VungleSDK/VungleSDKHeaderBidding.h>
```

Comment:
It' better to put `<VungleSDK/VungleSDKHeaderBidding.h>` before `#import "VungleInstanceMediationSettings.h"` to follow MoPub code style guidance, thanks a lot. :)

Reply:
Resolved.